pennylane/qchem/integrals.py

```diff
@@ -306,8 +307,8 @@ def _overlap_integral(*args):
             na
             * nb
             * (
-                (ca[:, anp.newaxis] * cb)
-                * gaussian_overlap(basis_a.l, basis_b.l, ra, rb, alpha[:, anp.newaxis], beta)
+                (ca[:, np.newaxis] * cb)
```

Comment:
Not the scope of this PR but why is it necessary to add some extra dimension?

Reply:
It helps to create different combinations of the orbitals without having deep `for` loops.